conda_libmamba_solver/index.py

```diff
@@ -409,3 +409,10 @@ def _process_query_result(
 
 # for conda-build
 _CachedLibMambaIndexHelper = lru_cache(maxsize=None)(LibMambaIndexHelper)
+
+
+# conda-build needs to operate offline for the index
+def _LibMambaIndexHelperOffline(*args, **kwargs):
+    with context._override("offline", True):
```

Comment:
Not sure if we want to use the more common `with env_var()` here instead of this surgical `_override`. Should be ok for simple booleans.

Reply:
I went for `env_var` + `reset_context` because I think if there's a reset_context call in the context manager scope, then the override is missed. With a new env var is not.